content/docs/contributing/to-docs/style-guide/media.md

```diff
@@ -129,3 +129,24 @@ Animated images are not currently accepted for contribution due to limitations o
 
 1. **Left-click** on the entity in **Perspective** to select it.
 1. **LMB+Drag** on the transform gizmo's **Z** axis to move the entity on the world **Z** axis.
+
+
+## Embedding Youtube videos with the `youtube-width` shortcode
+
+Embed Youtube videos in your page by using the `youtube-width` shortcode. The `youtube-width` shortcode is an extended version of Hugo's built-in [`youtube` shortcode](https://gohugo.io/content-management/shortcodes/#youtube) that allows you to control the size of the embedded video. 
+
+`youtube-width` takes three double-quoted parameters in order:
```

Comment:
Removed "in order" because it doesn't need to be. Also keeping as `youtube-width`. 
```suggestion
`youtube-width` supports the following double-quoted parameters:
```